misc/deployer/build.gradle

```diff
@@ -0,0 +1,233 @@
+}
+
+task unzipJars(type: Copy, dependsOn: ['zipJars', 'cleanup']) {
+	def path = project.buildDir.absolutePath + '/../../../' + '/zip/target/packages-'
+    def zipFile = file( path + version + '-SNAPSHOT/bundles.zip')
+    def outputDir = file(project.ext.deployfolder)
+    from zipTree(zipFile)
+    into outputDir
+}
+
+task getAllJars() << {
+    allprojects.each {
+        if (!it.extensions.extraProperties.has('allJars')) {
+            project.ext.allJars.add(it.configurations.archives.artifacts.files.files)
+        }
+
+    }
+    subprojects.each {
+        if (it.extensions.extraProperties.has('allJars')) {
+            project.ext.allJars.add(it.extensions.extraProperties.allJars)
+        }
+    }
+}
+
+getAllJars { outputs.upToDateWhen { false } }
+
+task deployDevClearCache(dependsOn: 'unzipJars', group: "Environment deployment", description: "Performs deployment on vagrant machine with karafs restart and removing cache") << {
+    ssh.run {
+        session(remotes.vagrant) {
+            execute 'sudo /etc/init.d/karaf stop', ignoreError: true
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/cache/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/generated-bundles/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/data/tmp/*'
+            execute 'rm -rf /opt/aet/karaf/current/data/log/*'
+            execute 'sudo rm -rf /opt/aet/karaf/current/deploy/*'
+            put from: fileTree(project.ext.deployfolder), into: '/opt/aet/karaf/current/deploy'
+            put from: file(project.buildDir.absolutePath + '/../../../' + '/zip/target/aet-features.xml'), into: '/opt/aet/karaf/current/deploy'
+            execute 'sudo /etc/init.d/karaf start'
+        }
+    }
+}
+
+task deployReport(group: "Environment deployment", description: "Deploy report") << {
+    ssh.run {
+        session(remotes.vagrant) {
+            execute 'sudo /etc/init.d/karaf stop'
+            execute 'sudo rm -rf /opt/aet/apache/aet_reports/current/*'
+            put from: file(project.buildDir.absolutePath + '/../../../' + '/report/src/main/webapp/report.html'), into: '/opt/aet/apache/aet_reports/current'
+            put from: (project.buildDir.absolutePath + '/../../../' + '/report/src/main/webapp/app'), into: '/opt/aet/apache/aet_reports/current'
+            put from: (project.buildDir.absolutePath + '/../../../' + '/report/src/main/webapp/assets'), into: '/opt/aet/apache/aet_reports/current'
+            put from: file(project.buildDir.absolutePath + '/../../../' + '/report/src/main/webapp/html/index.html'), into: '/opt/aet/apache/aet_reports/current'
+            execute 'sudo /etc/init.d/karaf start'
+        }
+    }
+}
+
+task devClearCache(group: "Environment deployment", description: "Clears karafs cache on vagrant") << {
+    ssh.run {
+        session(remotes.vagrant) {
+            execute 'sudo /etc/init.d/karaf stop'
+            execute 'sudo rm -rf /content/karaf/current/data/*'
```

Comment:
If we want those scripts to be in GitHub repository they need to correspond with what AET provide via Vagrant. I can't see another option, sorry...